app/views/shared/_draft_tools.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="button-list is-gutterless">
+  <a href="javascript:void(0)"
+     class="button is-muted is-outlined is-icon-only js-save-draft"
+     aria-label="Save draft"
+     title="Save draft"
+     role="button">
+    <i class="fas fa-save"></i>
+  </a>
+  <a href="javascript:void(0)"
+     class="button is-muted is-outlined is-icon-only js-delete-draft"
+     aria-label="Delete draft"
```

Comment:
Let's change "Delete draft" to "Delete saved draft" for a little extra clarity (we're not changing anything in this textbox, just what's saved for you).